docs/breaking-changes.md

```diff
@@ -12,14 +12,58 @@ This document uses the following convention to categorize breaking changes:
 * **Deprecated:** An API was marked as deprecated. The API will continue to function, but will emit a deprecation warning, and will be removed in a future release.
 * **Removed:** An API or feature was removed, and is no longer supported by Electron.
 
+## Planned Breaking API Changes (23.0)
+
+### Removed: `webContents.incrementCapturerCount(stayHidden, stayAwake)`
```

Comment:
Generally we also add a `Deprecated` section in the version the API becomes deprecated. You can pretty much just copy/paste this but replace 'removed' with 'deprecated'.